drivers/asr-driver/asr-driver-google/src/main/java/org/restcomm/media/drivers/asr/driver/google/GoogleAsrDriver.java

```diff
@@ -70,133 +71,129 @@
-        this.languages.add("ro-RO"); //Romanian (Romania)
-        this.languages.add("si-LK"); //Sinhala (Srilanka)
-        this.languages.add("sk-SK"); //Slovak (Slovakia)
-        this.languages.add("sl-SI"); //Slovenian (Slovenia)
-        this.languages.add("su-ID"); //Sundanese (Indonesia)
-        this.languages.add("sw-TZ"); //Swahili (Tanzania)
-        this.languages.add("sw-KE"); //Swahili (Kenya)
-        this.languages.add("fi-FI"); //Finnish (Finland)
-        this.languages.add("sv-SE"); //Swedish (Sweden)
-        this.languages.add("ta-IN"); //Tamil (India)
-        this.languages.add("ta-SG"); //Tamil (Singapore)
-        this.languages.add("ta-LK"); //Tamil (Sri Lanka)
-        this.languages.add("ta-MY"); //Tamil (Malaysia)
-        this.languages.add("te-IN"); //Telugu (India)
-        this.languages.add("vi-VN"); //Vietnamese (Vietnam)
-        this.languages.add("tr-TR"); //Turkish (Turkey)
-        this.languages.add("ur-PK"); //Urdu (Pakistan)
-        this.languages.add("ur-IN"); //Urdu (India)
-        this.languages.add("el-GR"); //Greek (Greece)
-        this.languages.add("bg-BG"); //Bulgarian (Bulgaria)
-        this.languages.add("ru-RU"); //Russian (Russia)
-        this.languages.add("sr-RS"); //Serbian (Serbia)
-        this.languages.add("uk-UA"); //Ukrainian (Ukraine)
-        this.languages.add("he-IL"); //Hebrew (Israel)
-        this.languages.add("ar-IL"); //Arabic (Israel)
-        this.languages.add("ar-JO"); //Arabic (Jordan)
-        this.languages.add("ar-AE"); //Arabic (United Arab Emirates)
-        this.languages.add("ar-BH"); //Arabic (Bahrain)
-        this.languages.add("ar-DZ"); //Arabic (Algeria)
-        this.languages.add("ar-SA"); //Arabic (Saudi Arabia)
-        this.languages.add("ar-IQ"); //Arabic (Iraq)
-        this.languages.add("ar-KW"); //Arabic (Kuwait)
-        this.languages.add("ar-MA"); //Arabic (Morocco)
-        this.languages.add("ar-TN"); //Arabic (Tunisia)
-        this.languages.add("ar-OM"); //Arabic (Oman)
-        this.languages.add("ar-PS"); //Arabic (State of Palestine)
-        this.languages.add("ar-QA"); //Arabic (Qatar)
-        this.languages.add("ar-LB"); //Arabic (Lebanon)
-        this.languages.add("ar-EG"); //Arabic (Egypt)
-        this.languages.add("fa-IR"); //Persian (Iran)
-        this.languages.add("hi-IN"); //Hindi (India)
-        this.languages.add("th-TH"); //Thai (Thailand)
-        this.languages.add("ko-KR"); //Korean (South Korea)
-        this.languages.add("cmn-Hant-TW"); //Chinese, Mandarin (Traditional, Taiwan)
-        this.languages.add("yue-Hant-HK"); //Chinese, Cantonese (Traditional, Hong Kong)
-        this.languages.add("ja-JP"); //Japanese (Japan)
-        this.languages.add("cmn-Hans-HK"); //Chinese, Mandarin (Simplified, Hong Kong)
-        this.languages.add("cmn-Hans-CN"); //Chinese, Mandarin (Simplified, China)
+        GoogleAsrDriver.LANGUAGES.add("af-ZA"); //Afrikaans (South Africa)
```

Comment:
this logic cannot be inside the constructor. Please use a 
```
static {
    LANGUAGES.add("af-ZA"); //Afrikaans (South Africa)
    LANGUAGES.add("am-ET"); //Amharic (Ethiopia)
    ...
}
```

block outside of the constructor.

Reply:
done.